pkg/agent/config_test.go

```diff
@@ -1,15 +1,202 @@
 package agent
 
 import (
+	"bytes"
 	"fmt"
+	"io"
+	"log"
+	"os"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/d4l3k/messagediff"
```

Comment:
**Self-review:** I'd be in favor of removing this dependency. Even if it's just a testing dep, it isn't needed: we can use testify's Equal which already gives us a good diff.

Reply:
I've removed this dependency in 21e52292fc0601cbc9a7af18ee500b0384d385b2.